benchmarks/nel/scripts/train.sh

```diff
@@ -1,15 +1,20 @@
 #!/bin/bash
 
-[[ -z "${4}" ]] && gpu_config="" || gpu_config="--gpu-id $4"
+[[ -z "${5}" ]] && gpu_config="" || gpu_config="--gpu-id $5"
```

Comment:
```suggestion
gpu_id="${5:--1}"
```

Without a GPU argument spaCy will run on CPU, which is equivalent to `--gpu-id -1`, so maybe this is clearer. Note this would also require a change below.